### DISCUSSION TITLE
Is application running sandboxed if GUI is closed? (No orange outline)

### DISCUSSION THREAD

**Discussion by K-Hilman** [sandboxie-plus/Sandboxie#1727] [Documents and notes]
Hi, I've recently created a shortcut for a programme installed in Sandboxie via "DefaultBox/Box Content/Create Shortcut". I've placed the shortcut on my desktop. I've noticed that, if I exit sandbox by right-clicking its icon at the windows tray, then clicking exit, I am still able to launch the application via the shortcut at my desktop. However, the usual orange outline will no longer be visible. 

Despite me exiting the Sandboxie app and no orange outline appearing, is the application still running via Sandboxie and is isolated?

**Comment by isaak654** [comment]
The application is still isolated if you used the shortcut generated with Sandboxie. See also #1340

If you want to restore the usual sandbox border with Sandboxie Plus, you can check the option "Start UI when a sandboxed process is started" in Options-> Global Settings -> Shell Integration.

**Reply by K-Hilman** [reply]
Thank you for the confirmation
